src/main/kotlin/org/ozwillo/dcimporter/config/ScheduledTask.kt

```diff
@@ -0,0 +1,17 @@
+package org.ozwillo.dcimporter.config
+
+import org.ozwillo.dcimporter.service.MarcheSecuriseListingService
+import org.springframework.context.annotation.Profile
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Component
+
+@Component
+@Profile("dev")
```

Comment:
Not only. Later we'll have preprod and prod profiles.

Maybe you want a @Profile("!test") ?

Reply:
Exactly, updated